driver/razerkbd_driver.c

```diff
@@ -2355,6 +2361,25 @@ static ssize_t razer_attr_write_matrix_effect_starlight(struct device *dev, stru
         }
         break;
 
+    case USB_DEVICE_ID_RAZER_BLADE_16_2025:
+        if(count == 7) {
+            request = razer_chroma_standard_matrix_effect_starlight_dual(buf[0], (struct razer_rgb*)&buf[1], (struct razer_rgb*)&buf[4]);
+            request.transaction_id.id = 0xFF;
+            razer_send_payload(device, &request, &response);
+        } else if(count == 4) {
+            request = razer_chroma_standard_matrix_effect_starlight_single(buf[0], (struct razer_rgb*)&buf[1]);
+            request.transaction_id.id = 0xFF;
+            razer_send_payload(device, &request, &response);
+        } else if(count == 1) {
+            request = razer_chroma_standard_matrix_effect_starlight_random(buf[0]);
+            request.transaction_id.id = 0xFF;
+            razer_send_payload(device, &request, &response);
+        } else {
+            printk(KERN_WARNING "razerkbd: Starlight only accepts Speed (1byte). Speed, RGB (4byte). Speed, RGB, RGB (7byte)\n");
+            return -EINVAL;
+        }
+        break;
```

Comment:
This is the same as the above for e.g. `USB_DEVICE_ID_RAZER_BLADE_STEALTH_LATE_2017` or am I mistaken?